task_01.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Task 01"""
+
+
+import decimal
+ABSOLUTE_DIFFERENCE = decimal.Decimal(('{:.2f}'.format(273.15)))
+
+
+def fahrenheit_to_celsius(degrees):
+    """a function called ``fahrenheit_to_celsius()`` that has one parameter:
```

Comment:
Please review function documentation guidelines -- this doesn't follow the template we put forward in class.